starport/cmd/validation.go

```diff
@@ -0,0 +1,28 @@
+package starportcmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/placeholder"
+)
+
+// FlagNoValidation disables validation.
+const FlagNoValidation = "no-validation"
```

Comment:
This is about only generating code without modification?

I think it should be renamed into something like `--generate-only` or `--no-modification`.
`--no-validation` sounds like to me like if we still modify the files but we don't check if the placeholders exist